index.html

```diff
@@ -40,6 +40,128 @@ <h1 class="my-name">I’m Mariane <br> Glad to see <br> you!</h1>
+                    </div>
+                    <p class="work-desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="categories">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button class="btn-primary">See Project</button>
+                </div>
+            </div>
+            <div class="work-card">
+                <div class="work-img">
+                    <img src="img/printing.png" width="275" alt="">
+                </div>
+                <div class="work-details">
+                    <h2>Professional Printing Works</h2>
+                    <div class="experience">
+                        <h4 class="title">Print</h4>
+                        <p class="role"> <img src="/img/Counter.png" alt=""> Back End Dev</p>
+                        <p class="year"> <img src="/img/Counter.png" alt=""> 2016</p>
+                    </div>
+                    <p class="work-desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="categories">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button class="btn-primary">See Project</button>
+                </div>
+            </div>
+        </section>
+
+        <section id="about">
+            <div class="about-content">
+                <h2>About <br>myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <div>
+                    <ul class="social-media-icon">
+                        <li><img src="/img/social icons/twiter icon.svg" alt=""></li>
+                        <li><img src="/img/social icons/github icon.svg" alt=""></li>
+                        <li><img src="/img/social icons/icon.svg" alt=""></li>
+                        <li><img src="/img/social icons/Linkedin icon.png" alt=""></li>
+                        <li><img src="/img/social icons/medium icon.svg" alt=""></li>
+                    </ul>
+                    <a href="#" class="connect-two">LET'S CONNECT</a>
+                </div>
+                <br>
+                <button class="btn-primary">Get my resume</button>
+                <div class="modal">
+                    <ul class="modal-items">
+                        <li class=".modal-item">
+                            <h3 class="item-one">Languages <img src="/img/Icon - Chevron - Down - Larger.png" alt=""></h3> <br>
+                            <span class="item"> <img src="/img/Ellipse 1.png" alt=""><h4>Javascript</h4></span> <br>                 
```

Comment:
- [ ] **\[OPTIONAL\]** I highly recommend avoid spaces in the file names.
`ellipse_1.png` _is better than_ `Ellipse 1.png`